openedx_learning/rest_api/v1/components.py

```diff
@@ -7,8 +7,11 @@
 
 
 class ComponentViewSet(viewsets.ViewSet):
+    """
+    Example endpoints for a Components REST API. Not implemented.
+    """
     def list(self, request):
-        items = Component.objects.all()
+        items = Component.objects.all()  # pylint: disable=unused-variable
```

Comment:
Nit: remove pylint comment
```suggestion
        _items = Component.objects.all()
```